### PR TITLE
Detail what functions need to be implemented

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

**Comment by edi33416** [comment]
Published at https://dlang-upb.github.io/Test-PCLP4-GH-Pages/refs/pull/4/merge

File: docs/project/milestone-1.md [hunk 1]
```diff
@@ -148,4 +148,13 @@ users.update(["_id" : "unique_id"],
 
 ### Tasks
 
-You will have to implement the functions marked with `TODO` in `Project/m1/db_conn/app.d`.
+You will have to implement the functions marked with `TODO` in `Project/m1/db_conn/app.d`:
+
+* `addUser - receives a users credentials and adds them to the user database
```
Comment (edi33416) [inline docs/project/milestone-1.md:153]: Please detail how a password should be stored: hash(gen salt + pass)

**Review by edi33416 — CHANGES_REQUESTED** [review]

File: docs/project/milestone-1.md (hunk 1 above)
Comment (RazvanN7) [reply docs/project/milestone-1.md:153]: Didn't we say this is a task for m2?

Comment (edi33416) [reply docs/project/milestone-1.md:153]: Oh. Ok. It wasn’t clear for me that that was the decision. I’ll merge this then

**Pull request merged**